chipsec/modules/common/secureboot/variables.py

```diff
@@ -63,44 +65,48 @@ def __init__(self):
         BaseModule.__init__(self)
         self._uefi = UEFI(self.cs)
 
-    def is_supported(self):
+    def is_supported(self) -> bool:
         supported = self.cs.helper.EFI_supported()
         if not supported:
-            self.logger.log_important("OS does not support UEFI Runtime API.  Skipping module.")
+            self.logger.log_important('OS does not support UEFI Runtime API.  Skipping module.')
             self.res = ModuleResult.NOTAPPLICABLE
         return supported
 
-    def can_modify(self, name, guid, data, attrs):
-        self.logger.log("    > Attempting to modify variable {}:{}".format(guid, name))
+    def can_modify(self, name: str, guid: Optional[AnyStr], data: Optional[bytes]) -> bool:
+        if not guid or not data:
+            self.logger.log(f'    > Attempting to modify variable {guid}:{name}')
+            return False
+        else:
+            self.logger.log(f'    > Attempting to modify variable {guid}:{name}')
```

Comment:
This log is duplicated, were they meant to be different?

Reply:
Good catch, thanks!